### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
##Summary
add footer

##Screenshot
captura de pantalla 2018-09-12 a la s 14 54 03

##Trello card
https://trello.com/c/BqVclOYk/14-footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,66 +8,77 @@
...
+    <h1 class="title">Front-End Tools</h1>
+    <div class="container-technologies">
+      <div class="techno-content">
+        <img class="img-techno" src="images/react_icon.svg" alt="React icon"/>
+        <p class="tech-text">ReactJS</p>
+        <p class="text-release">Release:2013</p>
+        <p class="text-complete"> React is a declarative,component-based Javascript library for building user interfaces.</p> 
+      </div>
+      <div class="techno-content">
+      <img class="img-techno" src="images/angular_icon.svg" alt="React icon"/>
```
Comment (mmamaniwolox) [inline index.html]: Indentation here

File: index.html [hunk 2]
```diff
@@ -8,66 +8,77 @@
...
-    <div class="techno-content">
-      <img class="img-techno" src="images/aspnet_icon.png" alt="React icon">
+      <img class="img-techno" src="images/aspnet_icon.png" alt="React icon"/>
       <p class="tech-text">VueJS</p>
       <p class="text-release">Release:2014</p>
       <p class="text-complete">ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.</p> 
     </div>
     <div class="techno-content">
-      <img class="img-techno" src="images/django_icon.svg" alt="React icon">
+      <img class="img-techno" src="images/django_icon.svg" alt="React icon"/>
```
Comment (mmamaniwolox) [inline index.html]: You have problems with indentation in this file, check it please

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -75,5 +75,10 @@ <h1 class="title">Back-End Tools</h1>
       </div>
     </div>
   </div>  
+  <footer class= "footer">
+    <p class="text-changes">WE ARE GAME CHANGERS</p>
+    <p class="text-wolox">W O L O X</p>
```
Comment (mmamaniwolox) [inline index.html]: I think there is an asset for this

File: scss/tech.scss [hunk 4]
```diff
@@ -1,5 +1,6 @@
  @import 'color';
 
+
```
Comment (mmamaniwolox) [inline scss/tech.scss]: delete empty line

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

File: index.html [hunk 5]
```diff
@@ -75,5 +75,10 @@ <h1 class="title">Back-End Tools</h1>
       </div>
     </div>
   </div>  
+  <footer class= "footer">
```
Comment (FrankIglesias) [inline index.html]: delete space after equals

File: index.html [hunk 6]
```diff
@@ -75,5 +75,10 @@ <h1 class="title">Back-End Tools</h1>
...
+    <p class="text-changes">WE ARE GAME CHANGERS</p>
+    <img class="img-footer" src="images/wolox_logo_white.svg" alt="Wolox icon"/>
```
Comment (FrankIglesias) [inline index.html]: change alt name

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request closed without merging**